### ISSUE TITLE
Module switch menu is not based on the installed modules list

### ISSUE THREAD

**Issue by jeffnm** [wc-library/Coral#27]
While testing the upgrade with a missing configuration file, I installed skipped the Licensing module (because I had deleted the configuration.ini file). All the other modules installed fine, but the module switch list in the menu still had the licensing module as an option. Clicking it results in either a not found or a 500 error. Ideally a module only appears in the Change Module list when it's installed according to the common/configuration.ini file.

**Comment by jcuenod** [comment]
hmm, I think this is implemented on a per module basis. I can look into changing these though.

**Comment by jeffnm** [comment]
@jcuenod I think you are right. This is probably not something you should worry about at this point, but rather something that we'll fix in the next release when we continue to combine common elements. That said, if you want to take a stab at it now, that's fine too, but I can't guarantee we'll get the result into the 2.0 beta release.

**Comment by jcuenod** [comment]
I don't think it will be hard - I suspect it will actually simplify code and just be a matter of `require_once` my common `Config.php` and then just query it. That's already happening in `index.php` but yes, getting this change into 2.0 is not essential in my mind it would be good to fix that though.
